pom.xml

```diff
@@ -13,7 +13,7 @@
   <artifactId>gwt-jsonix-schema-compiler</artifactId>
   <packaging>jar</packaging>
   <name>Jsonix GWT Schema Compiler</name>
-  <version>1.2.1-SNAPSHOT</version>
+  <version>1.2.2-SNAPSHOT</version>
```

Comment:
@danielzhe Last version release is `1.2.0`. The new version which we will release with your changes will be `1.2.1`. Therefore, please restore the previous version `1.2.1-SNAPSHOT`. When your PR will be merged, we will release tag version `1.2.1`. Please refer to the process we follow with 1.2.0. release (last 3 commits in the repo)

Reply:
Thank you for the clarification, @yesamer ! Changes applied.